presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/WriteConcernType.java

```diff
@@ -31,6 +31,7 @@
     SAFE(WriteConcern.SAFE),
     UNACKNOWLEDGED(WriteConcern.UNACKNOWLEDGED);
 
+    @SuppressWarnings("ImmutableEnumChecker")
     private final WriteConcern writeConcern;
```

Comment:
We should mark `WriteConcern` as immutable

Reply:
The `Immutable` can only be annotated in class level, and the class `WriteConcern ` is from 3rd party jar.